lib/local-links-manager/check_links/link_status_updater.rb

```diff
@@ -3,29 +3,27 @@
 module LocalLinksManager
   module CheckLinks
     class LinkStatusUpdater
```

Comment:
Small change should be made to the commit message so it starts with an active verb.